### PR TITLE
add /faq command.  Includes ESLint fixes

### PR DESCRIPTION
**Pull request by Zadag**
- [x] You have read the [Odin Bot README/Contributing Guide](https://github.com/TheOdinProject/odin-bot-v2/blob/master/README.md).

#### 1.Describe the changes made:

Adds faq.js faq.test.js and fixes linting errors in simpleCommands.test.js

#### 2. If this PR modifies a command, please provide a screenshot of the passing tests below. NOTE: Your PR will not be merged if you have any failing test cases. 

image

#### 3. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#132 #116

### CODE REVIEW

File: botCommands/faq.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    ['/fq'],
+    ['/f a q'],
+    ['```function("/faq", () => {}```'],
+    ['/FAQ'],
+    [''],
+    [' '],
+    ['/ '],
+    ['@odin-bot / faq'],
+    ['/fafaq'],
+    ['/bigNum'],
```
Comment (twalton83) [inline botCommands/faq.test.js:27]: Let's remove the `/bigNum` test case, we mostly want to test for edge cases at risk of triggering the regex

File: botCommands/simpleCommands.test.js [hunk 2]
```diff
@@ -840,9 +840,9 @@ describe('/xy', () => {
       ['/xy @odin-bot'],
       ['@odin-bot /xy'],
     ])('correct strings trigger the callback', (string) => {
-      expect(commands.xy.regex.test(string)).toBeTruthy()
-    })
-    
+      expect(commands.xy.regex.test(string)).toBeTruthy();
+    });
```
Comment (twalton83) [inline botCommands/simpleCommands.test.js:844]: Thanks for bundling this in.

**Review by twalton83 — CHANGES_REQUESTED** [review]

File: botCommands/faq.test.js (hunk 1 above)
Comment (Zadag) [reply botCommands/faq.test.js:27]: On it

**Review by Zadag — COMMENTED** [review]
Removed here

**Review by twalton83 — APPROVED** [review]
Thank you for fixing.
